build.gradle

```diff
@@ -26,12 +26,23 @@ buildscript {
 }
 
 plugins {
+    id 'com.gradle.build-scan' version '1.9'
     id "com.jfrog.bintray" version "1.7.3"
     id "com.jfrog.artifactory" version "4.4.10"
     id "nebula.optional-base" version "3.1.0"
     id "nebula.provided-base" version "3.1.0"
 }
 
+buildScan {
+    licenseAgreementUrl = 'https://gradle.com/terms-of-service'
+    licenseAgree = 'yes'
+    link 'GitHub', 'https://github.com/cbeust/testng/tree/master'
+    if (System.getenv('CI')) {
+        publishOnFailure()
```

Comment:
I think we can remove the test output now

Reply:
Fixed and pushed forcibly to the same commit. Please check